pkg/arvo/app/publish.hoon

```diff
@@ -1390,6 +1420,13 @@
   ::
   ==
 ::
+++  quit-collection
+  |=  wir=wire
+  ^-  (quip move _this)
+  =/  pax=path  (weld /collection wir)
+  :_  this
+  [ost.bol %peer pax [src.bol %publish] pax]~
```

Comment:
If your in-app convention is to always use a `wire` identical to the subscription `path`, you could get away with implementing a more generic `++quit` and not thinking about it in the future. But, right now the only subscriptions you do are `/collection/*` ones, so it doesn't matter much.

You could do `[%collection wir]` instead of `weld`ing. But that, too, doesn't matter very much.